### ISSUE TITLE
[3.0] Display the item that reached the limit (objectId is too big ... contact us)

### ISSUE THREAD

**Issue by tristanbes** [algolia/search-bundle#160]
It would be nice to display the row that reached the limit to give us insights on what is really too big.
Not a priority though.

<img width="1030" alt="1__app_seek-team___srv_app__ssh_" src="https://user-images.githubusercontent.com/346010/33600561-e672aeea-d9a9-11e7-9a43-19a3603fadb8.png">

**Comment by tristanbes** [comment]
Also, I think it could be nice to **continue** indexing the other rows and at the end, display the objectId that was in error, and why, using a table helper.

What do you think ?

**Comment by hhmmrrhh** [comment]
Open

**Comment by nunomaduro** [comment]
@tristanbes I don't think we should `continue indexing the other rows and at the end`, at least at the moment - as I consider this to be a `RuntimeException`.

But yeah, I would be great to have access to the doctrine entity information where this happened.

**Comment by tristanbes** [comment]
I continue to think that throwing an exception is not what we can exept.

I just did some maintenance operation on my website which consist in 3 operations:
- detect if users put some link in their description
- remove found links
- send emails

I got 50,000 rows to process.

At some point, the command crashes because of 1 item too big for Algolia. (Note that i'm using the Listeners from Doctrine)

So in a developper point of view:
- I don't know which items from my DB were processed
- Emails were sent
- I didn't use algolia in the code. Just calling `persist` and `flush`

I'm using DoctrineBatchUtils so I can't wrap the persist/flush in a try/catch condition to catch Algolia exception.

Now I need to find out why this row is too big, and then launch the command again, risking sending 2x the same email to users.

Hope you can reconsider with this example.
